nuxt.config.ts

```diff
@@ -79,6 +81,17 @@ export default defineNuxtConfig({
           return `${code}\nexport default { render: render }`
         },
       },
+      sentryVitePlugin({
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        org: 'sentry',
+        project: 'aescan-develop',
+        url: 'https://sentry.dev.service.aepps.com/',
+      }),
+      sentryVitePlugin({
+        org: 'sentry',
```

Comment:
Is it correct that there's no `authToken` here?

Reply:
Right, fixed